src/error.rs

```diff
@@ -165,6 +165,8 @@ pub enum ValidationErrorKind {
     RecordMagicByte,
     /// Index blob size
     IndexBlobSize,
+    /// Index is not written (index header corrupted)
+    IndexNotWritten,
```

Comment:
There is `IndexIsWritten` error above. It seems to me that it was mistakenly used for cases where the index is not actually written.  Please, check all places where `IndexIsWritten` is generated. If you find a place where its usage is incorrect, replace it with the new `IndexNotWritten`

Reply:
Done